### ISSUE TITLE
The BOT Unfollow and Follow Pattern

### ISSUE THREAD

**Issue by saabtech** [mifi/SimpleInstaBot#200]
I guess it will be better if the BOT unfollow  random number of accounts and then followed by following random Accounts to look it bit more human.

At the moment, the bot Unfollow straight out 66 accounts and then move to Following. looks pretty much straight forward pattern to a BOT.

**Comment by ediooo** [comment]
Yes, I am getting action blocked almost every hour now, with just a daily limit of following 27 and very few likes.

**Comment by ediooo** [comment]
The follows and unfollows should also never be happening "in blocks" of just one action, they should be randomly mixed. 3 follows then 5 unfollows then 1 follow etc.

**Comment by saabtech** [comment]
> Yes, I am getting action blocked almost every hour now, with just a daily limit of following 27 and very few likes.

When that happen, Just logout and give the account a long break , say 2 days and then login back. make sure the account you are using on BOT is not loggedin in any device if so do logout all devices and give it a break.
